tests/Unit/Css/CssDocumentTest.php

```diff
@@ -488,6 +525,32 @@ public function notRendersInvalidNonConditionalAtRule(string $atRuleCss, string
         self::assertStringNotContainsString($atAndRuleName, $result);
     }
 
+    /**
+     * @test
+     *
+     * @param string $atRuleCss
+     * @param string $cssBefore
+     *
+     * @dataProvider provideInvalidNonConditionalAtRule
+     */
+    public function throwsExceptionForOrDiscardsInvalidNonConditionalAtRuleInDebugMode(
+        string $atRuleCss,
+        string $cssBefore = ''
+    ): void {
+        try {
+            $subject = self::createDebugSubject($cssBefore . $atRuleCss);
+
+            $result = $subject->renderNonConditionalAtRules();
+
+            \preg_match('/@[\\w\\-]++/', $atRuleCss, $atAndRuleNameMatches);
+            $atAndRuleName = $atAndRuleNameMatches[0];
+            self::assertStringNotContainsString($atAndRuleName, $result);
+        } catch (UnexpectedTokenException $exception) {
+            // Passed test
+            self::expectNotToPerformAssertions();
```

Comment:
Ditto about testing only one case (and possibly having the other case in a separate, deactivated, test).

Reply:
Ditto.